client/coral-admin/src/containers/ModerationQueue/ModerationContainer.js

```diff
@@ -70,8 +70,10 @@ class ModerationContainer extends Component {
       <div>
         <ModerationHeader asset={asset} />
         <ModerationMenu
-          activeTab={activeTab}
           asset={asset}
+          premodCount={data.premod.length}
```

Comment:
Not this kind of count. This needs to come back from the server. Otherwise this will mess up the pagination.